src/client.js

```diff
@@ -79,6 +80,27 @@ module.exports = class Client {
     })
   }
 
+  // Maintains a pool of static endpoints with auto-batching enabled
+  autoBatch (endpointName, batchDelay) {
```

Comment:
@queicherius When I was designing this, I wasn't sure of a cleaner way to "toggle on" autobatching with a similar syntax to the rest of the library, something like `client.autobatch().items()` .  Do you have some thoughts on a good way to do that?  I'd like to keep the style of the library consistent.

Reply:
as a second note, my implementation doesn't support child endpoints, like account().achievements().  The primary use case I'm imagining is items(), but I'd like it to just be a feature everywhere that it could be useful.

Reply:
Actually I have an idea for how to do this with an autobatchPool class returned from client.autoBatch() that would function as a wrapper for the endpoints... This should be a lot cleaner

Reply:
Right now, it just gets passed down (e.g. here's how language works):
- Set language on the client: https://github.com/queicherius/gw2api-client/blob/master/src/client.js#L25-L29
- Inherit the language from the parent (this can be a client or a other endpoint): https://github.com/queicherius/gw2api-client/blob/master/src/endpoint.js#L12
- Set the language on the endpoint: https://github.com/queicherius/gw2api-client/blob/master/src/endpoint.js#L44-L48